packages/adblocker-puppeteer/test/adblocker.test.ts

```diff
@@ -36,3 +37,34 @@ describe('#fromPuppeteerDetails', () => {
     });
   });
 });
+
+describe('#stylesInjection', () => {
+
+  it('does not inject styles into original content', async () => {
+    const browser = await puppeteer.launch();
+    try {
+      const page = await browser.newPage();
+      try {
+        const url = 'https://example.com';
+        const stylesInjectionPrefix = '<style type="text/css">#Meebo\\:AdElement\\.Root';
```

Comment:
Not sure if this will be stable over time as filters get updated. It might be safer to initialize the `blocker` instance with a set of fake rules for the sake of this test.